plugins/base/src/main/resources/dokka/scripts/platform-content-handler.js

```diff
@@ -31,12 +31,16 @@ window.addEventListener('load', () => {
 const darkModeSwitch = () => {
     const localStorageKey = "dokka-dark-mode"
     const storage = localStorage.getItem(localStorageKey)
-    const savedDarkMode = storage ? JSON.parse(storage) : false
+    const osDarkSchemePreferred = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches
+    const savedDarkMode = storage ? JSON.parse(storage) : osDarkSchemePreferred
```

Comment:
I think `savedDarkMode` can be renamed to something like `darkModeEnabled` since fallback value is `osDarkSchemePreferred`, and OS theme is not really "saved"

Reply:
Agree, I'll rename it.